### ISSUE TITLE
bTable: fixed not working because b-table-fixed class is missing from css

### ISSUE THREAD

**Issue by 2-5** [bootstrap-vue/bootstrap-vue#2690]
rc13: when using `fixed` on `bTable` the `b-table-fixed` class is correctly added to the generated `<table>` element, but it doesn't work, because `b-table-fixed` is missing from the CSS, both in the precompiled one (`bootstrap-vue.min.css`) or when including it throught `.scss`

In the CSS we only have this relevant class, which as you can see, it's not called `b-table-fixed`:

```css
.b-table.table.table-fixed {
  /* fixed width columns */
  table-layout: fixed;
}
```

**Comment by jackmu95** [comment]
Closed by #2676.

**Comment by jackmu95** [comment]
@2-5 Will be fixed in the next RC.
